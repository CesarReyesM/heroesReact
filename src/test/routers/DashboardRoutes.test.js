
import React from 'react';
import { mount } from "enzyme";
import { AuthContext } from "../../auth/AuthContext";
import { DashboardRoutes } from "../../routers/DashboardRoutes";
import '../../setupTest';
import { MemoryRouter } from 'react-router';



describe('pruebas en el dashboard', () => {

    const contextValue = {
        dispatch : jest.fn(),
        user: {
            logged: false
        }
    }

   
    test('primera prueba', () => {
        const wrapper = mount(
            <AuthContext.Provider value= {contextValue}>
                <MemoryRouter>
                    <DashboardRoutes/>
                </MemoryRouter>
            </AuthContext.Provider>
        )
        expect(wrapper).toMatchSnapshot();

    })


})


