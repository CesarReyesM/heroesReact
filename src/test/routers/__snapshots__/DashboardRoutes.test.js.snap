// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en el dashboard primera prueba 1`] = `
<Fragment>
  <Navbar />
  <div
    className="container mt-2"
  >
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/marvel"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/hero/:heroeId"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/dc"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/search"
      />
      <Redirect
        to="/marvel"
      />
    </Switch>
  </div>
</Fragment>
`;
